### PR TITLE
CVE-2020-1753 init

### PR DESCRIPTION
**Pull request by rehakm**

### CODE REVIEW

File: 2020/1xxx/CVE-2020-1753.json [hunk 1]
```diff
@@ -4,15 +4,90 @@
...
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "all Ansible 2.7.x versions prior to 2.7.17"
+                                        },
+                                        {
+                                            "version_value": "all Ansible 2.8.x versions prior to 2.8.11"
+                                        },
+                                        {
+                                            "version_value": "all Ansible 2.8.x versions prior to 2.9.7"
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1753.json]: Shouldn't it be `2.9.x` instead of `2.8.x`?

File: 2020/1xxx/CVE-2020-1753.json [hunk 2]
```diff
@@ -4,15 +4,90 @@
...
+                "refsource": "CONFIRM"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "A security flaw was found in Ansible Engine, all Ansible 2.7.x versions prior to 2.7.17, all Ansible 2.7.x versions prior to 2.8.11 and all Ansible 2.7.x versions prior to 2.9.7, when managing kubernetes using the k8s module. Sensitive parameters such as passwords and tokens are passed to kubectl from the command line, not using an environment variable or an input configuration file. This will disclose passwords and tokens from process list and no_log directive from debug module would not have any effect making these secrets being disclosed on stdout and log files."
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1753.json]: I think there are copypasto in the versions, i.e. the second `2.7.x` should be `2.8.x` and the third `2.7.x` should be `2.9.x`.

File: 2020/1xxx/CVE-2020-1753.json [hunk 3]
```diff
@@ -4,15 +4,90 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1753",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1753",
+                "refsource": "CONFIRM"
+            }
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1753.json]: Please add a reference to upstream pull request:

 <https://github.com/ansible-collections/kubernetes/pull/51>

Comment (rehakm) [reply 2020/1xxx/CVE-2020-1753.json]: Such a mess, I'm so sorry. Fixed everything.

File: 2020/1xxx/CVE-2020-1753.json (hunk 1 above)
Comment (bmillemathias-1a) [inline 2020/1xxx/CVE-2020-1753.json]: all Ansible 2.9.x

Comment (iamleot) [reply 2020/1xxx/CVE-2020-1753.json]: That seems was already addressed in commit https://github.com/CVEProject/cvelist/pull/3406/commits/684171ad7cf8ac067938f7f5a2e325e97dd54fa2 (i.e. no further changes are needed.)

**Pull request merged**